spec/azure/loadbalancer/loadbalancer.go

```diff
@@ -0,0 +1,10 @@
+package loadbalancer
```

Comment:
This should sit in virtualnetwork package

Reply:
Ack, fix is in https://github.com/giantswarm/azuretpr/pull/4